signbank/dictionary/adminviews.py

```diff
@@ -200,13 +200,13 @@ def get_queryset(self):
 
 
         # A list of phonology fieldnames
-        fieldnames = ['handedness', 'strong_handshape', 'weak_handshape', 'location', 'relation_between_articulators',
+        """  fieldnames = ['handedness', 'strong_handshape', 'weak_handshape', 'location', 'relation_between_articulators',
```

Comment:
Please just remove code if it's unused, rather than commenting it out - it diffs better with git, and it is a nice way to reflect the code is actually unused

Reply:
My mistake, I missed it! It should have been deleted.